src/h5web/vis-packs/core/shared/Html.tsx

```diff
@@ -70,20 +85,32 @@ const Html = forwardRef<HTMLDivElement, HtmlProps>((props, ref) => {
       if (parentElement) {
         el.remove();
       }
+
       ReactDOM.unmountComponentAtNode(el);
     };
-  }, [el, parentElement]); // eslint-disable-line react-hooks/exhaustive-deps
+  }, [el, parentElement]);
 
+  // Update size and overflow of container `div`
   useLayoutEffect(() => {
+    el.style.width = `${width}px`;
+    el.style.height = `${height}px`;
+    el.style.overflow = followCamera ? 'hidden' : 'visible';
+  }, [el, width, height, followCamera]);
+
+  // Render inner `div`
+  useEffect(() => {
+    group.current.updateMatrixWorld(); // make sure we get the group's most up to date position
```

Comment:
`updateMatrixWorld` is now called here, before rendering the inner `div` (instead of before appending the outer `div`.

Note that this is now a `useEffect` instead of `useLayoutEffect`. This is because `useFrame` can now be safely called before this `useEffect`.